conftest.py

```diff
@@ -187,6 +187,13 @@ def pytest_configure(config):
             "filterwarnings",
             r"ignore:datetime\.datetime\.utcfromtimestamp\(\) is deprecated:DeprecationWarning",
         )
+        # On Python 3.12+, there is a deprecation warning when calling os.fork()
+        # in a multi-threaded process
```

Comment:
this comment should be a condition. Something like
`if find_spec("ratarmount") is not None and sys.version_info >= (3, 12)`: